inductiva/simulators/dualsphysics.py

```diff
@@ -19,6 +19,8 @@ def run(
         commands: List[dict],
         machine_group: Optional[resources.MachineGroup] = None,
         storage_dir: Optional[types.Path] = "",
+        save_sim_metadata_dir: Optional[types.Path] = None,
+        extra_sim_metadata_to_save: Optional[dict] = None,
```

Comment:
for now let's not have any additional argument to these methods.
Let's assume that they get called with everything that is strictly required. If it becomes imperative to have additional, custom, metadata fed into these calls, then we'll look into it.